### ISSUE TITLE
Merge resolution is not working (Meld Diff Error: Scm diff type 17 not supported)

### ISSUE THREAD

**Issue by iameatingbits** [danielroedl/vsc-meld-diff#23]
It seems that something has changed internally in vscode in part of handling git scm related operations -- so now vscode does send 
new type code of value **17**.

My setup:

OS:
Edition	Windows 11 Pro
Version	22H2
Installed on	‎01.‎04.‎2023
OS build	22621.1778
Experience	Windows Feature Experience Pack 1000.22642.1000.0

VS code:
Version: 1.79.0 (user setup)
Commit: b380da4ef1ee00e224a15c1d4d9793e27c2b6302
Date: 2023-06-07T14:26:35.552Z
Electron: 22.5.5
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621

Using meld ext along with win/wsl/remote configs of vscode.

Below is the patch I have crafted asap and tested -- it is working fine and as expected.
```
diff --git a/src/extension.ts b/src/extension.ts
index 1e42174..b89a927 100644
--- a/src/extension.ts
+++ b/src/extension.ts
@@ -537,6 +537,7 @@ export function activate(context: vscode.ExtensionContext) {
                                break;

                     case 16: // merge conflicts
+                    case 17: // new type of conflict
                                // get content of head version of the selected file
                                runGit(selectedFile, ":2:./", "current", (head, err) => {
                                        if (err) {
```

**Comment by iameatingbits** [comment]
@danielroedl hello ^^^

**Comment by danielroedl** [comment]
Hi @iameatingbits 
thanks for the bug report and the patch. I am currently on vacation.  So I think I can release a new version with your fix tomorrow.

**Comment by danielroedl** [comment]
Change is published with version 1.4.1. Thanks for your support!

**Comment by iameatingbits** [comment]
@danielroedl great! ty
